Tasks/DownloadPackageV0/task.json

```diff
@@ -82,7 +82,7 @@
                 "feed": "$(feed)",
                 "definition": "$(definition)"
             },
-            "endpointUrl": "{{endpoint.url}}/{{ #regex ([a-fA-F0-9\\-]*/)[a-fA-F0-9\\-]+ feed }}_apis/Packaging/Feeds/{{ #regex [a-fA-F0-9\\-]*/([a-fA-F0-9\\-]+) feed }}{{#if view}}@{{{view}}}{{/if}}/Packages/{{{definition}}}/Versions?includeUrls=false&protocolType={{{packageType}}}",
+            "endpointUrl": "{{endpoint.url}}/{{ #regex ^([a-fA-F0-9\\-]+\/)[a-fA-F0-9\\-]+$ feed }}_apis/Packaging/Feeds/{{ #regex ^[a-fA-F0-9\\-]*?\/?([a-fA-F0-9\\-]+)$ feed }}{{#if view}}@{{{view}}}{{/if}}/Packages/{{{definition}}}/Versions?includeUrls=false",
```

Comment:
Why are we changing this regex?

Reply:
The original definition had the feed id in the format /<feedId> where it was a non project scoped feed. I changed that definition to just be <feedId>, as we have updated the release management tasks as well to just have <feedId>. The new regex takes into account the missing / at the beginning of the feedId